test/end-to-end.js

```diff
@@ -134,6 +134,25 @@ describe('end-to-end', function() {
     })
   })
 
+  it('should normalize strings using NFC', function(done) {
+    var req = coap.request({
+        port: port
+        // U+210E (plank constant) becomes to U+0068 in “compatible” normalizations (should not happen)
+        // U+0065 (e) U+0301 (combining acute accent) becomes U+00e9 in “composed” normalizations (should happen)
+      , pathname: '/\u210e/\u0065\u0301'
+    }).end()
+
+    server.on('request', function(req, res) {
+      try {
+        expect(req.url).to.equal('/\u210e/\u00e9')
+        done()
+        res.end()
+      } catch (e) {
+        done(e)
+      }
+    })
+  })
```

Comment:
Forgot to remove it completely (used it to fix the test at some point). Will be out in next commit.